readme.md

```diff
@@ -81,27 +81,54 @@ to [run the E2E test](./LOCAL_TESTING.md).
 [Discord](https://discord.com/invite/zetachain) |
 [Telegram](https://t.me/zetachainofficial) | [Website](https://zetachain.com)
 
+## Releases
 
-## Creating a Release Candidate
-Creating a release candidate for testing is a straightforward process. Here are the steps to follow:
+To start a new major release, begin by creating and pushing a `release/` branch based on `develop`.
 
-### Steps
- - Step 1. Create the release candidate tag with the following format (e.g., vx.x.x-rc) ex. v11.0.0-rc.
- - Step 2. Once a RC branch is created the automation will kickoff to build and upload the release and its binaries.
+<details>
+<summary>Example Commands</summary>
+
+```bash
+git fetch
+git checkout -b release/v15 origin/develop
+git push origin release/v15
+```
+
+</details>
+
+Most changes should first be merged into `develop` then backported to the release branch via a PR.
```

Comment:
It's generally good to backport the change to the release branch via a PR so you can make sure CI is still passing. 

I've previously used [this github action](https://github.com/marketplace/actions/backporting) to automatically generate a backport PR when the develop PR is merged. All you have to do is apply a `backport release/v15` label to your develop PR.

But we could also allow developers to just directly cherry pick and push their changes to the release branch and run the CI on push instead.